4.Clients/WebApi/ClientApp/src/app/login/csoft-signin.component.ts

```diff
@@ -12,59 +12,55 @@ import { Router } from '@angular/router';
 })
 export class CSoftComponent {
 
-    loginForm: FormGroup;
-    authenticatedUser:User;
-    submitForm(): void {
-      for (const i in this.loginForm.controls) {
-        this.loginForm.controls[i].markAsDirty();
-        this.loginForm.controls[i].updateValueAndValidity();
-      }
+  loginForm: FormGroup;
+  authenticatedUser:User;
 
-      if (this.loginForm.valid) {
-        this.authenticateUser(this.loginForm.controls.userName.value, this.loginForm.controls.password.value);        
-      }
-      
-    }
-  
-    constructor(private fb: FormBuilder, private facade: FacadeService, private jwtHelper: JwtHelper, private router: Router, public zone: NgZone) {}
-  
-    ngOnInit(): void {
-      this.loginForm = this.fb.group({
-        userName: [null, [Validators.required]],
-        password: [null, [Validators.required]]
-      });
-    }
+  constructor(private fb: FormBuilder, private facade: FacadeService, private jwtHelper: JwtHelper, private router: Router, public zone: NgZone) {}
```

Comment:
Cuando son muchos parametros en un constructor o en un metodo ponelos uno debajo del otro así se pueden ver más facil.
```suggestion
  constructor(
     private fb: FormBuilder,
     private facade: FacadeService, 
     private jwtHelper: JwtHelper, 
     private router: Router, 
     public zone: NgZone
  ) {}
```